### ISSUE TITLE
Erroneously reporting incorrect order

### ISSUE THREAD

**Issue by gajus** [jacobrask/eslint-plugin-sorting#7]
```js
properties: {
    dateChangeCount: definitions.count,
    eventTimeViewCount: definitions.count,
    locationAddressChangeCount: definitions.count,
    locationListActivateCount: definitions.count,
    locationMapActivateCount: definitions.count,
    locationMapChangeCount: definitions.count,
    trailerViewCount: definitions.count
},
```

It is telling me this block is not sorted alphabetically.

**Comment by gajus** [comment]
Just in case, the entire block is:

```js
definitions.eventCount = {
    additionalProperties: false,
    descripition: 'Records the number of events that occurred to this moment.',
    title: 'history',
    type: 'object',
    properties: {
        dateChangeCount: definitions.count,
        eventTimeViewCount: definitions.count,
        locationAddressChangeCount: definitions.count,
        locationListActivateCount: definitions.count,
        locationMapActivateCount: definitions.count,
        locationMapChangeCount: definitions.count,
        trailerViewCount: definitions.count
    },
    required: [
        'dateChangeCount',
        'eventTimeViewCount',
        'locationAddressChangeCount',
        'locationListActivateCount',
        'locationMapActivateCount',
        'locationMapChangeCount',
        'trailerViewCount'
    ]
};
```

**Comment by gajus** [comment]
Never mind. Forgot alphabet.
